src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component.html

```diff
@@ -39,20 +59,20 @@
 <ng-template #readOnlyFeedback>
     <div class="container">
         <div class="row align-items-center">
```

Comment:
```suggestion
        <div class="row align-items-center m-0">
```
to get rid of the negative margin that got put there by row